From: deeksha.cs <deeksha.cs@ltts.com>
Date: Thu Apr 23 09:39:05 2020 +0000

Subject: REFPLTB-560,REFPLTB-162: wan-status sysevent not changing to "stopped" on RPI,
                         Removing the wan cable and putting it back is not bringing erouter0 up

Reason for change: wan-status sysevent not showing down when ethernet is disconnected - Rules for erouter and all_lans added in rt_tables.
                   wan-status sysevent not showing up when ethernet is connected, operstate link was down. Fixed the issues
Test Procedure: Connect Ethernet cable to RPi. Check operstate file in image - should show up. sysevent get wan-status should show started.
                Disconnect Ethernet cable to RPi. Check operstate file in image - should show down. sysevent get wan-status should show stopped
Risks:Low

Signed-off-by: deeksha.cs <deeksha.cs@ltts.com>
---

--- iproute2-4.7.0-old/etc/iproute2/rt_tables	2016-08-08 15:58:39.000000000 +0000
+++ iproute2-4.7.0/etc/iproute2/rt_tables	2020-04-22 14:04:42.384605093 +0000
@@ -1,3 +1,5 @@
+3 erouter
+4 all_lans
 #
 # reserved values
 #
